auth/keycloak/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # >docker build -t keycloak .
 # >docker run --env-file=../../.env -d keycloak
 # https://hub.docker.com/r/jboss/keycloak/
-FROM jboss/keycloak:11.0.0
+FROM jboss/keycloak:16.1.1
```

Comment:
I'm fine with this but shouldn't this warrant it's own PR in case we need to revert?

Reply:
It could, I was considering this low risk since this is a PR to dev that affects local dev only.